ci/README.md

```diff
@@ -195,7 +195,7 @@ jobs:
         uses: actions/checkout@v4
       - name: Read nightly version
         id: read_toolchain
-        run: echo "nightly_version=$(cat nightly-version)" >> $GITHUB_OUTPUT
+        run: echo "nightly_version=$(cat nightly_version)" >> $GITHUB_OUTPUT
```

Comment:
This is wrong, the file is called `nightly-version`.

Reply:
Ok then line 181 must be wrong. I'll change it.

https://github.com/rust-bitcoin/rust-bitcoin-maintainer-tools/pull/15/files#diff-794891fcb80950b0c81f330c416efdf5d35789fb56a91a18dcc25266d7cdccfdR181